modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpServerTransport.java

```diff
@@ -300,7 +299,6 @@ protected HttpChannelHandler(final Netty4HttpServerTransport transport, final Ht
         protected void initChannel(Channel ch) throws Exception {
             Netty4HttpChannel nettyHttpChannel = new Netty4HttpChannel(ch);
             ch.attr(HTTP_CHANNEL_KEY).set(nettyHttpChannel);
-            ch.pipeline().addLast("chunked_writer", new Netty4WriteThrottlingHandler(transport.getThreadPool().getThreadContext()));
```

Comment:
@henningandersen I thought about this a little more and I don't see the point in double-throttling outside of the no_split case. I think the no_split case is sufficiently fringe to ignore here and not optimize for. 

=> dropped the low level throttling in favor of just having the high level throttling now for the REST path.